tzrec/main.py

```diff
@@ -737,6 +739,8 @@ def _script_model(
             logger.info("quantize embeddings...")
             quantize_embeddings(model, dtype=torch.qint8, inplace=True)
 
+        if is_aot():
+            model = model.cuda()
```

Comment:
just model.cuda() is correct？when i use gloo to load device_state_dict = state_dict_to_device(
            model.state_dict(), pg=checkpoint_pg, device=torch.device("cpu")
        )
model = model.to_empty(device="cpu")
model = model.to("cuda:0") may be incorrect when I run forward model(data_cuda)

Reply:
I have test it, there is no problem